### DISCUSSION TITLE
https://r-leyshon.github.io/quarto-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [r-leyshon/quarto-giscus#5] [Announcements]
# https://r-leyshon.github.io/quarto-giscus/

https://r-leyshon.github.io/quarto-giscus/

**Comment by r-leyshon** [comment]
# Testing

~~markdown~~ **functionality**

[some link](https://www.google.com.mx/)

![some image](https://i.imgur.com/MiCmAtN.jpeg)

Testing revisions from within the GitHub discussion.

**Reply by r-leyshon** [reply]
Responding to a comment
